rubato/c_src/cdraw.cpp

```diff
@@ -47,39 +46,37 @@ PIXEL FUNCTIONS
 inline void setPixel(size_t _pixels, int width, int height, int x, int y, size_t color, bool blending = false) {
     if (x < width && y < height && x >= 0 && y >= 0) {
         int off = y * width + x;
-        uint32_t added = (uint32_t) color;
-        uint32_t* pixels = (uint32_t*) _pixels;
+        uint_fast32_t added = (uint_fast32_t) color;
+        uint_fast32_t* pixels = (uint_fast32_t*) _pixels;
```

Comment:
this is wrong. uint_fast32_t is `at least 32 bits`, and you need `exactly 32 bits`. that is the only difference.